cylc/flow/scripts/broadcast.py

```diff
@@ -409,7 +409,3 @@ def main(_, options: 'Values', workflow: str) -> None:
             print(get_broadcast_change_report(
                 modified_settings, is_cancel=report_cancel))
     sys.exit(report_bad_options(bad_options, is_set=report_set))
-
-
-if __name__ == "__main__":
-    main()
```

Comment:
Why is this no longer needed? Is it because all commands are mapped to in `setup.cfg`?

Reply:
Yes, this allows the commands to be run by executing the files directly which was never intended.